lib/uwb/protocols/fira/UwbConfiguration.cxx

```diff
@@ -18,6 +18,116 @@ const std::unordered_set<ResultReportConfiguration> UwbConfiguration::ResultRepo
+         return config->GetPreambleCodeIndex();
+     } },
+    { uwb::protocol::fira::UwbApplicationConfigurationParameterType::RangingTimeStruct, [](UwbConfiguration* config) -> std::optional<uwb::protocol::fira::UwbApplicationConfigurationParameterValue> {
+         return config->GetRangingTimeStruct();
+     } },
+    { uwb::protocol::fira::UwbApplicationConfigurationParameterType::SlotsPerRangingRound, [](UwbConfiguration* config) -> std::optional<uwb::protocol::fira::UwbApplicationConfigurationParameterValue> {
+         return config->GetSlotsPerRangingRound();
+     } },
+    { uwb::protocol::fira::UwbApplicationConfigurationParameterType::PrfMode, [](UwbConfiguration* config) -> std::optional<uwb::protocol::fira::UwbApplicationConfigurationParameterValue> {
+         return config->GetPrfMode();
+     } },
+    { uwb::protocol::fira::UwbApplicationConfigurationParameterType::ScheduledMode, [](UwbConfiguration* config) -> std::optional<uwb::protocol::fira::UwbApplicationConfigurationParameterValue> {
+         return config->GetSchedulingMode();
+     } },
+    { uwb::protocol::fira::UwbApplicationConfigurationParameterType::KeyRotationRate, [](UwbConfiguration* config) -> std::optional<uwb::protocol::fira::UwbApplicationConfigurationParameterValue> {
+         return config->GetKeyRotationRate();
+     } },
+    { uwb::protocol::fira::UwbApplicationConfigurationParameterType::MacAddressMode, [](UwbConfiguration* config) -> std::optional<uwb::protocol::fira::UwbApplicationConfigurationParameterValue> {
+         return config->GetMacAddressMode();
+     } },
+    { uwb::protocol::fira::UwbApplicationConfigurationParameterType::MaxRangingRoundRetry, [](UwbConfiguration* config) -> std::optional<uwb::protocol::fira::UwbApplicationConfigurationParameterValue> {
+         return config->GetMaxRangingRoundRetry();
+     } },
+    { uwb::protocol::fira::UwbApplicationConfigurationParameterType::UwbInitiationTime, [](UwbConfiguration* config) -> std::optional<uwb::protocol::fira::UwbApplicationConfigurationParameterValue> {
+         return config->GetUwbInitiationTime();
+     } },
+    { uwb::protocol::fira::UwbApplicationConfigurationParameterType::HoppingMode, [](UwbConfiguration* config) -> std::optional<uwb::protocol::fira::UwbApplicationConfigurationParameterValue> {
+         return config->GetHoppingMode();
+     } },
+    // { uwb::protocol::fira::UwbApplicationConfigurationParameterType::ResultReportConfig, [](UwbConfiguration* config) -> std::optional<uwb::protocol::fira::UwbApplicationConfigurationParameterValue> {
+    //      return config->GetResultReportConfigurations();
+    //  } },
+
+    // params with different names
+    // { uwb::protocol::fira::UwbApplicationConfigurationParameterType::RangingRoundUsage, [](UwbConfiguration* config) -> std::optional<uwb::protocol::fira::UwbApplicationConfigurationParameterValue> {
```

Comment:
Why is this one commented out? I believe that RangingRoundUsage (UCI) and RangingMethod (CSML) correspond to the same exact thing, so this should be fine to include here

Reply:
There's an issue with the neutral and DDI types, so I commented them out for now.